Formula/s/serveit.rb

```diff
@@ -11,8 +11,25 @@ class Serveit < Formula
     sha256 cellar: :any_skip_relocation, all: "e17a21fd5705de1b2e96656ad703f51c8f1781601f295ff7028cfc859bd0ca44"
   end
 
+  depends_on "ruby"
```

Comment:
Does `uses_from_macos "ruby"` not work?

Reply:
it wont work for linux build as it would use ruby 3.3, so I just switched all builds to use ruby 3.3 as result

Reply:
can we keep `uses_from_macos` and just scope the gem resource to `on_linux`?

Reply:
yeah, I can try that.